### ISSUE TITLE
SVG object loading

### ISSUE THREAD

**Issue by thipages** [WebReflection/uhtml#14]
Hi,
I tried to render an interactive svg with the following code
the issue : it loops (n loaded on console, blinking on screen).
Thank you.
I can write a code pen if needed.

```javascript
const dispatch=(id, detail)=>()=>document.dispatchEvent(new CustomEvent(id, {detail:detail}))
const svg_test=(id,path)=> html`
        <object
            type="image/svg+xml"
            .data=${path}
            onload=${dispatch(id)}
        </object>
`;
const m= {
    id:'_svg_',
    path:'a valid path'
}
const update=()=> {
    render(document.body, svg_test(m.id, m.path));
}
document.addEventListener(m.id, ()=> {
    console.log("loaded");
    // do some work here like changing the svgs height/scale
    update();
});
update();
```

**Comment by WebReflection** [comment]
your `svg_test` is broken ... it doesn't close the initial object.

would this work instead?

```js
const svg_test=(id,path)=> html`
        <object
            type="image/svg+xml"
            .data=${path}
            onload=${dispatch(id)}
        />
```

**Comment by thipages** [comment]
Sorry, bad copy/paste.
same behaviour with the code you propose.
I write a code pen

**Comment by thipages** [comment]
here it is : https://codepen.io/thipages/pen/XWXKgBw

**Comment by WebReflection** [comment]
OK, this is an expected behavior, see the following code that uses zero libraries:

```js
document.body.innerHTML = `<object height="100px" type="image/svg+xml"></object>`;
const object = document.body.firstElementChild;
const m= {
  id: '_svg_',
  path: 'https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/ruby.svg'
};
const update = () => {
  object.onload = () => {
    document.dispatchEvent(new CustomEvent(m.id, {detail: null}));
  };
  object.data = m.path;
};
document.addEventListener(m.id, () => {
  setTimeout(()=> {
    console.log("loaded");
    // do some work here like changing the svgs height/scale
    update();
  },1000);
});
update();
```

setting the `.data` over and over makes it flick, but admittedly there was no way to pass `data` as attribute, but nodes that have a special data meaning should have the ability to use that indeed.

So, latest version of _uhtml_ allows you to use `data=${...}` instead of `.data=${...}` for nodes where `data` has a special meaning, so that the `dataset` helper is not triggered.

**TL;DR** this now works:

```js
const svg_test=(id,path)=> html`
  <object
    type="image/svg+xml"
    data=${path}
    onload=${dispatch(id)}
  />
`;
```

**Comment by thipages** [comment]
great! Thank you
